src/lib/components/ResultsPerPage/ResultsPerPage.js

```diff
@@ -49,13 +49,31 @@ export default class ResultsPerPage extends Component {
     this.updateQuerySize(value);
   };
 
+  renderSpanWithMargin = (text, margin) => {
+    const size = '0.5em';
+    let style;
+    switch (margin) {
+      case 'left':
+        style = { marginLeft: size };
+        break;
+      case 'right':
+        style = { marginRight: size };
+        break;
+      default:
+        style = { margin: `0 ${size}` };
+    }
+    return <span style={style}>{text}</span>;
+  };
+
   render() {
-    const { loading, currentSize, totalResults } = this.props;
+    const { loading, currentSize, totalResults, prefix, suffix } = this.props;
```

Comment:
May I suggest a change that IMHO it fits a bit better the purpose of the prefix/suffix props without making it too specific? As I understood, please correct me if I am wrong, the end goal is to apply a custom label that wraps the ResultsPerPage component. I would go then for a label prop that can be a function like this: 
```javascript
ResultsPerPage label={(cmp) => `Show {cmp} results per page`} />
```

then in the render function of the `ResultsPerPage` component you could write something similar to this:
```javascript

render() {
const { loading, currentSize, totalResults, label } = this.props;
...
{label && label(this.renderElement(currentSize, this.options, this.onChange))}
...
}
```